9.2/run-postgresql.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# For SCL enablement
+. /var/lib/pgsql/.bashrc
+
+set -eu
+
+# Data dir
+export PGDATA=/var/lib/pgsql/data
+
+# Be paranoid and stricter than we should be.
+psql_identifier_regex='^[a-zA-Z_][a-zA-Z0-9_]*$'
+psql_password_regex='^[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]+$'
+
+function usage {
```

Comment:
could you maybe do the function declaration in this file more consistent with the test/run script?